src/sema/namespace.rs

```diff
@@ -490,6 +507,106 @@ impl Namespace {
         }
     }
 
+    pub(super) fn resolve_error(
+        &mut self,
+        file_no: usize,
+        contract_no: Option<usize>,
+        path: &pt::IdentifierPath,
+        diagnostics: &mut Diagnostics,
+    ) -> Result<usize, ()> {
+        // if we are resolving an event name without namespace (so no explicit contract name
+        // or import symbol), then we should look both in the current contract and global scope
```

Comment:
I don't get what's wrong with the original text or what you are trying to improve.

The only problem I see is `event name` rather than `error name`.

Reply:
`look` here is ambiguous. I may mean `search` or `see`. If it means `see`, the preposition should be `at` not `in`.

Reply:
We mean `search`, not `see`

Reply:
Please, use `search` then, because it is preciser.

Reply:
I've changed to search, although I think the improvement is marginal.